### ISSUE TITLE
Don't start the ipfs gateway with Partyshare

### ISSUE THREAD

**Issue by bhstahl** [BusterLabs/Partyshare#19]
Since we're not using it at all.

**Comment by cvan** [comment]
@bhstahl first off, awesome project. you've made something tangible that's been in my head and notebooks for a while. kudos on this! 👍 🎉 

in a separate project that uses IPFS, I've noticed that files I've added to ipfs and pinned have expired. the daemon I start + shut down 30 seconds after publishing assets (i.e., a site).

just out of curiosity, have you encountered that issue? (I should probably be asking this in a separate issue, but this issue seemed related.)

thanks!

**Comment by bhstahl** [comment]
Thanks @cvan!

Just to confirm what you're asking, in a separate IPFS project you:

1. Add a file to IPFS
1. Retrieve it via a gateway
1. Stop your local daemon
1. Wait 30 seconds
1. Second retrieval of file via gateway fails

**Comment by jamiew** [comment]
And it sounds like you also pin the file, right?

**Comment by cvan** [comment]
@jamiew: yes, that's correct. haven't not looked too closely at your code and the supporting packages, is that a similar process you're doing?
